modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -151,13 +155,50 @@ def update_status!(status:, code: nil, detail: nil)
                                       statusable_id: id
                                     })
 
+      email_handler = Events::Handler.new(event_type: :nod_received, opts: {
+                                            email_identifier: email_identifier,
+                                            first_name: veteran_first_name,
+                                            date_submitted: veterans_local_time.iso8601,
+                                            guid: id
+                                          })
+
       update!(status: status, code: code, detail: detail)
 
       handler.handle!
+      email_handler.handle! if status == 'submitted' && email_identifier.present?
     end
 
     private
 
+    def mpi_veteran
```

Comment:
Not a blocker, but if these added methods are all the same for getting the email_identifier, maybe we should pull them out into a concern. A ticket for later, I'd assume :)